src/shared/components/create-case-filters/create-case-filters.component.spec.ts

```diff
@@ -17,6 +19,14 @@ const EVENT_NAME_2 = 'Event two';
 const EVENT_ID_3 = 'ID_3';
 const EVENT_NAME_3 = 'Event three';
 
+const ROLE1 = 'role1';
```

Comment:
I think it's unnecessary to create a new variable. This can be inline. ```createACL('role1', true, true, true, false);```

Reply:
done